src/run_app.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+curl -o src/get-pip.py https://bootstrap.pypa.io/pip/3.6/get-pip.py -o get-pip.py \
+&& apt-get update \
+&& apt-get install python3-apt \
+&& apt-get install python3.6-distutils -y \
+&& python3 src/get-pip.py \
+&& pip install toml \
```

Comment:
this one should goes to [here ](https://github.com/budtmo/docker-android/blob/master/requirements.txt)

Reply:
Thank you @budtmo, you are right. I moved the installation of the `toml` package to `requirements.txt`